lib/configure.js

```diff
@@ -439,15 +432,31 @@ module.exports = function() {
     if (!('after_success' in travisYml) || travisYml.after_success === null) {
       travisYml.after_success = [];
     }
-    command = 'echo "travis_fold:end:after_success" && ' +
-              '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ] && ' +
-              'echo "Deploying…" && gulp deploy';
-    if (!travisYml.after_success.some(function(v) { return v.indexOf(command) !== -1; })) {
-      process.stdout.write('Adding after_success command: ' + command + '…\n');
-      travisYml.after_success.push(command);
+
+    var defaultRemote = 'origin';
+    var remotes = childProcess.execSync('git remote').toString().split('\n').slice(0, -1);
+    if (remotes.indexOf('upstream') !== -1) {
+      defaultRemote = 'upstream';
+    }
+
+    var promise;
+    if (remotes.length === 1) {
+      promise = Promise.resolve(remotes[0]);
+    } else {
+      promise = promptly.prompt('Remote repository [default: ' + defaultRemote + ']: ', { default: defaultRemote });
     }
 
-    writeYaml.sync('.travis.yml', travisYml);
+    return promise.then(function(remote) {
+      command = 'echo "travis_fold:end:after_success" && ' +
+                '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ] && ' +
+                'echo "Deploying…" && gulp deploy --remote ' + remote;
```

Comment:
Although the remote to deploy can be different on a user's local machine, it should always be *origin* on Travis, given how Travis clones the repo.